pkg/utils/resource/resource_test.go

```diff
@@ -51,3 +54,60 @@ func TestHashOf(t *testing.T) {
 		})
 	}
 }
+
+// TestIsObjOversized tests the IsObjOversized function.
+func TestIsObjOversized(t *testing.T) {
+	cm := &corev1.ConfigMap{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "v1",
+			Kind:       "ConfigMap",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "app",
+			Namespace: "default",
+		},
+		Data: map[string]string{
+			"key": "value",
+		},
+	}
+
+	testCases := []struct {
+		name               string
+		sizeLimitBytes     int
+		wantErred          bool
+		wantSizeDeltaBytes int
+	}{
+		{
+			name:               "positize size delta",
```

Comment:
can you call this "under size limit"?

This test name is "positive size delta" but the wantSizeDeltaBytes is a negative number